### PR TITLE
Move back and update CSS table of contents to global.css

### PR DESCRIPTION
**Pull request by benoitchantre**
## Description

Addresses issue #351.

## List of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:
- [x] This pull request relates to a ticket.
- [x] This code is tested.
- [ ] This change has been added to CHANGELOG.md
- [x] I want my code added to WP Rig.

### CODE REVIEW

**Comment by ataylorme** [comment]
@benoitchantre can you please update the PR to merge into `v2.0` instead of `master`?

File: assets/css/src/global.css [hunk 1]
```diff
@@ -1,3 +1,28 @@
+/*
+CSS normalization based in part on normalize.css by
```
Comment (ataylorme) [inline assets/css/src/global.css]: We should probably keep this comment in `reset.css`

**Comment by benoitchantre** [comment]
> @benoitchantre can you please update the PR to merge into `v2.0` instead of `master`?

Oops, sorry. Done.

File: assets/css/src/global.css (hunk 1 above)
Comment (benoitchantre) [reply assets/css/src/global.css]: Yes

**Pull request merged**